layouts/shortcodes/contact.html

```diff
@@ -1,25 +1,38 @@
 <head>
-    <link rel="stylesheet" href="/css/contact-styles.css">
+    <link rel="stylesheet" href="/css/styles.css">
 </head>
-<div id="contact-popup">
+<div id="contact-container">
     <form>
         <div class="form-group">
-            <label for="name">Name:</label>
-            <input type="text" id="name" name="name" required>
+            <label for="name" style=>Name</label>
+            <input type="text" id="name" name="name" placeholder="John Doe" required>
         </div>
         <div class="form-group">
-            <label for="email">Email:</label>
-            <input type="email" id="email" name="email" required>
+            <label for="email">Email</label>
+            <input type="email" id="email" name="email" placeholder="example@email.com" required>
         </div>
         <div class="form-group">
-            <label for="message">Message:</label>
-            <textarea id="message" name="message" required></textarea>
+            <label for="message">Message</label>
+            <textarea id="message" name="message" placeholder="Lorem ipsum dolor sit amet," required></textarea>
         </div>
-        <a
-            href="mailto:get_in_touch.with.me.slightly636@slmail.me?subject=Get in touch through the website&body=Name: [Name]%0D%0AEmail: [Email]%0D%0AMessage: [Message]">
-            <button type="submit">Submit</button>
-        </a>
+        <button type="submit">Submit</button>
     </form>
 </div>
+<script>
```

Comment:
Unless absolute path is given, scripts does not seem to work for the files that are in `shortcodes.html`